plugins/modules/route53_health_check.py

```diff
@@ -628,9 +628,9 @@ def main():
 
             else:
                 if update_delete_by_id:
-                    changed, action = update_health_check(existing_check)
+                    changed, action, check_id = update_health_check(existing_check)
                 else:
-                    changed, action = update_health_check(existing_check)
+                    changed, action, check_id = update_health_check(existing_check)
```

Comment:
Both the if and else parts are calling the same function with the same arguments. Am I missing something?

Reply:
good catch, I wonder how that did not got caught earlier, seems redundant, will remove, test and push.